src/main/java/de/fau/amos4/model/Client.java

```diff
@@ -22,7 +24,55 @@
     @Column
     String companyName;
 
-    public long getId()
+    @Column
+    Boolean activated;
+
+    @Column
+    String confirmationString;
+    
+    @Column
+    String email;
+
+    public String getEmail() {
+		return email;
+	}
+
+	public void setEmail(String email) {
+		this.email = email;
+	}
+
+	public Boolean getActivated() {
+		return activated;
+	}
+
+	public void setActivated(Boolean activated) {
+		this.activated = activated;
+	}
+
+	public String getConfirmationString() {
+		return confirmationString;
+	}
+
+	public void setConfirmationString(String confirmationString) {
+		this.confirmationString = confirmationString;
+	}
+	
+	public void generateConfirmationString() {
```

Comment:
Maybe. I think we are going to need other methods here for further Client related functionality - with these together, it would better to move all them together to a service class.